adcc/modified_transition_moments.py

```diff
@@ -72,10 +74,43 @@ def compute_modified_transition_moments(gs_or_matrix, dipole_operator,
                         "libadcc.OneParticleOperator.")
 
     if method.is_core_valence_separated:
-        raise NotImplementedError("compute_modified_transition_moments "
-                                  "not implemented for CVS")
-
+        if method.name == "cvs-adc1":
+            return AmplitudeVector(dipole_operator['o2v1'])
+        else:
+            cvs_mtm = Adc2CVSModifiedTransitionMoments(method,
+                                                       ground_state,
+                                                       dipole_operator)
+            return cvs_mtm.cvs_adc2_mtm()
```

Comment:
I don't see a good reason to make a class for this. Why don't you just implement it as free functions?